packages/react-router/docs/Switch.md

```diff
@@ -51,4 +51,17 @@ This is also useful for animated transitions since the matched `<Route>` is rend
 
 ## children: node
 
-All children of a `<Switch>` should be `<Route>` elements. Only the first child to match the current location will be rendered.
+All children of a `<Switch>` should be `<Route>` or `<Redirect>` elements. Only the first child to match the current location will be rendered.
+
+`<Route>` elements are matched using their `path` prop and `<Redirect>` elements are matched using their `from` prop. A `<Route>` with no `path` prop or a `<Redirect>` with no `from` prop will always match the current location.
```

Comment:
Good points 👍 
Personally, I'd find it simpler to only use and support `path` for both, but I understand that using the `from` / `to` couple was very tempting.